packages/vscode-extension/media/walkthroughs/get-started/create/create.md

```diff
@@ -1,8 +1,8 @@
 | Step summary        | Where it's located              |
 |----------|----------------|
-| 1. Click "Create a New App" | ![Image1](step1.png) |
-| 2. Select Bot to create a new bot project | ![Image2](step2.png) |
-| 3. Ensure that Chat Notification Message is selected as the App feature that you want to build in your app. | ![Image3](step3.svg) |
+| 1. Click "Create a New App". | ![Image1](step1.png) |
+| 2. Select Bot to create a new bot project. | ![Image2](step2.png) |
+| 3. Make sure your select Chat Notification Message as the App feature that you want to build in your app. | ![Image3](step3.svg) |
```

Comment:
change 'your' to 'you' i.e. Make sure you select...

Reply:
Updated. Thanks!